amato/umbrel-app.yml

```diff
@@ -0,0 +1,21 @@
+manifestVersion: 1
+id: amato
+category: productivity
+name: Amato Timer
+version: "0.0.1.1"
+tagline: Simple work timer.
+description: >-
+  A simple timer with default 25 minute increments to track your work.
+releaseNotes: ""
+developer: Bitparasol
+website: http://bitparasol.com
```

Comment:
Is this your company and website? I don't see anything related to Amato timer? These fields are here to provide additional info on the app to users who may be curious.

We should also add the following two fields:
```yaml
repo: <the-repository-where-the-code-lives-if-public>
support: <link-to-support-site-or-github-issues>
```

If you don't add a repo, then the UI will show your source code being private like this:
![image](https://github.com/getumbrel/umbrel-apps/assets/85373263/0a5f2a38-4ceb-4326-850a-c1d0adff6268)


See Core Lightning app as an example:
https://github.com/getumbrel/umbrel-apps/blob/776587169cba9e345fcbc9e54f82675fd10e9eb2/core-lightning/umbrel-app.yml#L19-L24